earthfile2llb/parser/EarthParser.g4

```diff
@@ -4,7 +4,7 @@ options {
 	tokenVocab = EarthLexer;
 }
 
-earthFile: NL* (stmts NL)? NL* targets? NL* EOF;
+earthFile: NL* (stmts NL)? NL* targets? NL*? EOF;
```

Comment:
`EOF` repeats as often as needed; so if it gets eaten in `NL` we get another to wrap the file